mlflow/onnx.py

```diff
@@ -182,10 +182,27 @@ def _load_model(model_file):
 
 
 class _OnnxModelWrapper:
-    def __init__(self, path):
+    def __init__(self, path, providers=None):
         import onnxruntime
 
-        self.rt = onnxruntime.InferenceSession(path)
+        if providers is None:
+            providers = ['CUDAExecutionProvider', 'CPUExecutionProvider']
+
+        # NOTE: Some distributions of onnxruntime require the specification of the providers argument on calling. E.g. onnxruntime-gpu.
```

Comment:
Please adjust line length to 100 chars max.

Reply:
Will gladly adjust in the morning and submit commits to update this.

Reply:
On new commit.
Text notes have been limited to 100 characters and rewritten to according to the altered implementation.